src/utils/common.js

```diff
@@ -0,0 +1,8 @@
+import moment from 'moment';
+
+const isCourseEnded = (endDate) => moment(endDate) < moment();
```

Comment:
how does moment differ from just using `Date.now`?

Reply:
I think for this use case they are equivalent ... ```It's similar to native JavaScript's new Date().``` borrowed from some doc I read.. but I think moment in general makes working with date and time easier, handles locales better etc.. not too relevant to this pr I guess but yeah ...